public/javascripts/index.js

```diff
@@ -1,19 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const ButtonChoices = React.createClass({
-    render() {
-	return (
-	    <div id="buttons">
-	    <div id="button-group">
-	    <div id="capital-button">A</div>
-	    <div id="lowercase-button">a</div>
-	    </div>
-	    </div>
-	)
```

Comment:
I think you already got feedback on this but it will be more readable and easier to keep track of if you indent the nested divs.  Maybe this is the way you do it in react though?  I haven't gotten there.  Looking forward to seeing the finished product!